src/HotChocolate/Fusion-vnext/src/Fusion.Composition/Logging/LogEntryHelper.cs

```diff
@@ -262,6 +262,22 @@ public static LogEntry OutputFieldTypesNotMergeable(
             schemaA);
     }
 
+    public static LogEntry ProvidesOnNonCompositeField(
+        OutputFieldDefinition externalField,
+        INamedTypeDefinition type,
+        SchemaDefinition schema)
+    {
+        var coordinate = new SchemaCoordinate(type.Name, externalField.Name);
+
+        return new LogEntry(
+            string.Format(LogEntryHelper_ProvidesOnNonCompositeField, externalField.Name, schema.Name),
```

Comment:
```suggestion
            string.Format(
                LogEntryHelper_ProvidesOnNonCompositeField,
                externalField.Name,
                schema.Name),
```

Let's aim to wrap at 100 characters, even if the EditorConfig file includes `rulers = 120`.